vue.config.js

```diff
@@ -28,8 +28,8 @@ module.exports = {
     devServer: {
       proxy: {
         "/api": {
-          // target: "https://orgbook.gov.bc.ca",
-          target: "http://host.docker.internal:8080",
+          target: "https://dev.orgbook.gov.bc.ca",
+          // target: "http://host.docker.internal:8080",
```

Comment:
Intentional change swapping these for merging to main?

Reply:
Plan was to add it during the migration but I see no reason not to add it now. So here it is

Reply:
I should just stick with the original local host option. My mistake